src/jaegertracing/reporters/Config.h

```diff
@@ -20,15 +20,18 @@
 #include <chrono>
 #include <memory>
 #include <string>
+#include <utility>
 
 #include "jaegertracing/Logging.h"
-#include "jaegertracing/UDPTransport.h"
+#include "jaegertracing/ThriftSender.h"
 #include "jaegertracing/metrics/Metrics.h"
 #include "jaegertracing/reporters/CompositeReporter.h"
 #include "jaegertracing/reporters/LoggingReporter.h"
 #include "jaegertracing/reporters/RemoteReporter.h"
 #include "jaegertracing/reporters/Reporter.h"
 #include "jaegertracing/utils/YAML.h"
+#include "jaegertracing/utils/HttpTransporter.h"
+#include "jaegertracing/utils/UDPTransporter.h"
```

Comment:
why case inconsistency? UPD vs. Http - look like different conventions (Go vs. Java).

Reply:
no reason, I found HTTP to be a bit cumbersome. will change it.

Reply:
renamed.